.travis.yml

```diff
@@ -155,7 +155,7 @@ script:
     if [[ -v HIVE_TESTS && -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
       env AWS_ACCESS_KEY_ID=$HIVE_TESTS_AWS_ACCESS_KEY_ID \
         AWS_SECRET_ACCESS_KEY=$HIVE_TESTS_AWS_SECRET_ACCESS_KEY \
-        ./mvnw -pl presto-hive test -P test-hive-glue
+        ./mvnw -pl presto-hive test -B -P test-hive-glue
```

Comment:
Sounds good, but I am curious -- what problem does it fix?

Reply:
It actually does not fix anything, it just makes the output less verbose (no progress is logged for artifacts downloading)

Reply:
Add this to the commit msg